packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -150,6 +157,7 @@ export default class QuanticResultQuickview extends LightningElement {
 
   closeQuickview() {
     this.isQuickviewOpen = false;
+    this.previewRendered = false;
```

Comment:
no need here

Reply:
but inside `openQuickview` you should add a `this.isFirstPreviewRender = true;`

Reply:
Changed to `this.isFirstPreviewRender = true`